Makefile

```diff
@@ -301,7 +305,7 @@ unit-test-coverage:
 
 .PHONY: vendor
 vendor:
-	$(GO) mod tidy && TAGS="$(TAGS) vendor" $(GO) mod vendor
+	$(GO) mod tidy && $(GO) mod vendor
```

Comment:
But then the packages on build.go will not be vendored.

Reply:
Read my comment above. go modules will account for all combinations of build tags.
Running `make vendor` after this change didn't change vendor files.